src/Resolver/OptionResolver.php

```diff
@@ -19,32 +21,40 @@ class OptionResolver implements OptionResolverInterface {
    */
   protected $entityTypeManager;
 
+  /**
+   * The commerce options field manager.
+   *
+   * @var \Drupal\commerce_option\ProductOptionFieldManager
+   */
+  protected $fieldManager;
+
   /**
    * Constructs a new OptionResolver object.
    *
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
    *   The entity manager.
+   * @param \Drupal\commerce_option\ProductOptionFieldManager $field_manager
+   *   The option field manager.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+  public function __construct(
+    EntityTypeManagerInterface $entity_type_manager,
```

Comment:
Do we need entity type manager now in this class?

Reply:
Yeah its not needed anymore. I have removed it now